test/pages/mailbox.page.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-undef */
+/* eslint-disable linebreak-style */
+/* eslint-disable class-methods-use-this */
+
+const Page = require('./page');
+

```

Comment:
I would suggest collecting all selectors in a separate object called selectors
e.g. const selectors = {
  loginButton: './/*[@id="ybar-inner-wrap"]/div[3]/div/div[3]/div[1]/div/a')',
   inputEmailField: '#login-username', 
}